From d59e76f83642176b234912199199048c7843dbdb Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 15:20:11 +0200
Subject: [PATCH 41/76] prefs: disable network/DNS prediction

Stop wasting my bandwidth (however little), and also don't tell others
what I am about to visit.
---
 chrome/browser/net/prediction_options.cc | 1 -
 chrome/browser/net/prediction_options.h  | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git chrome/browser/net/prediction_options.cc chrome/browser/net/prediction_options.cc
index 5268257b841fc..9282bbbcb4574 100644
--- chrome/browser/net/prediction_options.cc
+++ chrome/browser/net/prediction_options.cc
@@ -31,7 +31,6 @@ NetworkPredictionStatus CanPrefetchAndPrerender(
       }
       return NetworkPredictionStatus::DISABLED_DUE_TO_NETWORK;
     default:
-      DCHECK_EQ(NETWORK_PREDICTION_NEVER, network_prediction_options);
       return NetworkPredictionStatus::DISABLED_ALWAYS;
   }
 }
diff --git chrome/browser/net/prediction_options.h chrome/browser/net/prediction_options.h
index 2ae6a1093090d..7fc83de1aa0fa 100644
--- chrome/browser/net/prediction_options.h
+++ chrome/browser/net/prediction_options.h
@@ -22,7 +22,7 @@ enum NetworkPredictionOptions {
   NETWORK_PREDICTION_ALWAYS,
   NETWORK_PREDICTION_WIFI_ONLY,
   NETWORK_PREDICTION_NEVER,
-  NETWORK_PREDICTION_DEFAULT = NETWORK_PREDICTION_WIFI_ONLY,
+  NETWORK_PREDICTION_DEFAULT = NETWORK_PREDICTION_NEVER,
 };
 
 enum class NetworkPredictionStatus {
-- 
2.29.1

